README.md

```diff
@@ -1,32 +1,31 @@
 # Mensinator🩸
 Private period tracking with custom symptoms. We don't track your data - you do. Period.
 
-## 🫂 Made for friends by friends
-Originally Emma made Mensinator for her friend with goals of better symptom tracking and privacy respect.  
-Soon, many of us joined and are now actively building the Mensinator together.  
-It is friendly, straightforward, and respects privacy.  
-We wanted an app to use, not an app to sell data to random corporations.
-
 ## 💡 Features
-- Period and ovulation tracking and statistics,
-- ***Customizable and colour-coded symptoms***,
-- Cycle prediction with average period calculation,
-- Cycle prediction with average ***luteal phase calculation for irregular cycles***,
-- Data export and import for backups or moving to another device,
-- No tracking. Your data is yours. No data is sent anywhere.
-- No bloat. No useless or complex features. No sign-up. No emails.
+- Period and ovulation tracking and statistics.
+- ***Customizable, colour-coded symptoms***.
+- Cycle prediction with average period calculation.
+- Cycle prediction with average ***luteal phase calculation for irregular cycles***.
+- Data export and import for backups or moving to another device.
+- No tracking. No sign-up. Your data is yours. No data is sent anywhere.
+- No bloat. No useless or complex features. No emails.
+
+## 🫂 Made for friends by friends
+Originally, Emma made Mensinator for a friend who was looking for an app that would track unique period symptoms and respect her privacy.
+Soon, more like-minded individuals joined the project. 
+We all wanted a user-friendly, straigtforward app made for us, not an app that uses our data for profit. We are now actively building Mensinator together. 
 
-## 🔮 Future ideas
-- Whatever you suggest: Discuss ideas with users and developers on **[our Discord](https://discord.gg/tHA2k3bFRN)**. Let us know what'd be good to have.
+## 🔮 Future plans
+- Whatever you suggest! Discuss ideas with users and developers on **[our Discord](https://discord.gg/tHA2k3bFRN)**. Let us know what features you'd like to see.
 
 ## 🤝 Active development and user community
 Join **[our Discord](https://discord.gg/tHA2k3bFRN)** to share your thoughts, ask questions, and chat with other users and our team.  
-Mensinator is in early development, made by people like you. Your feedback will help us improve it.  
-Some of us even stick around to nag the developers to make the app into what we want it to be. And some of us develop the features directly.
+Mensinator is in early development, made by people like you. Your feedback helps us continue improving it.  
+Share a quick thought, or stick around to nag the developers to make the app into what you want it to be. You can even help by developing the features directly.
```

Comment:
I intended 'some of us' to reveal user-developer relation of more friendly cooperative nature.
New suggestion embraces more of a us-them relation, less personal perhaps.

Reply:
Makes sense. I see what you're going for here too... my wording might not be best either, but I thought the original was a little too casual. The balance between casual/friendly and still professional (which I think conveys trustworthiness in a way, like "we know what we're doing even though we're just some random people") is difficult. We can keep working on it.